<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iniciomesero</class>
 <widget class="QWidget" name="iniciomesero">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mesero</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonCrear">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear Pedido</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonEliminar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar Pedido</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonFacturar">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Facturar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
